modules/analytics/analytics.js

```diff
@@ -123,6 +123,13 @@ class Analytics {
                     permanentProperties.group = group;
                 }
 
+                if (window.jitsiRegionInfo) {
```

Comment:
@paweldomas +1 for the proposal about writing down the "special agreements" in the coding style .md. Great idea! If we don't start doing this probably most of the agreements will be forgotten!